tool/gravity/cli/run.go

```diff
@@ -438,9 +439,12 @@ func Execute(g *Application, cmd string, extraArgs []string) error {
 			*g.AppStatusCmd.Locator,
 			*g.AppStatusCmd.OpsCenterURL)
 	case g.AppUninstallCmd.FullCommand():
-		return uninstallApp(localEnv,
-			*g.AppUninstallCmd.Locator,
-			*g.AppUninstallCmd.OpsCenterURL)
+		var appPackage *loc.Locator
+		appPackage, err = loc.ParseLocator(*g.AppUninstallCmd.Locator)
```

Comment:
Won't this return an error right away if the locator isn't provided to the command? Which would render your changes for determining local app automatically above not having any effect?